imap_data_access/io.py

```diff
@@ -162,6 +164,7 @@ def upload(file_path: Union[Path, str]) -> None:
     file_path : pathlib.Path or str
         Path to the file to upload. It must be located within
         the ``imap_data_access.config["DATA_DIR"]`` directory.
+
```

Comment:
These newlines seem a little superfluous to me because they are still within the docstring 🤷 , I feel like I'd almost prefer the newline after this if anything.

Reply:
:joy: I mean I'd personally prefer that, so if you do too I think we should. We don't have to take every check, I think it is just getting it to something you like. I'm also fine leaving it in, it just stuck out to me here.